### DISCUSSION TITLE
My 2FA number has been blocked for trying too many times for OTP SMS Send / Resend.

### DISCUSSION THREAD

**Discussion by lutfullahillabib** [community/community#136939] [Other Feature Feedback, Questions, & Ideas]
Done.

**Comment by m3h3d1ha2an** [comment]
did you tried to send sms to your number  using another sim number ?

**Reply by lutfullahillabib** [reply]
No. Didn't use another mobile number. Used same as before. ( +8801911302979 )

**Comment by AbuBakkarRaihan** [comment]
@lutfullahillabib Bro If your number is blocked due to multiple OTP requests, please contact GitHub support directly through their [Contact Form](https://support.github.com/contact). Include the transaction details your phone number and the email associated with your account. They can help you solve the 2FA problem by opening your number.

**Reply by lutfullahillabib** [reply]
I tried this. It's nothing but fixed chat bot. No way to contact or send email GitHub support directly. or explain my problem clearly.

**Comment by 0nahid** [comment]
@lutfullahillabib follow this [link](https://support.github.com/contact/account?contact[type]=account&flow=2fa)

**Reply by lutfullahillabib** [reply]
Tried.. its nothing but fixed chatbot..!! 😥

**Comment by AbdullahWins** [comment]
you must contact github support and explain why you have requested otp repeatedly (network issue/didn't get otp etc)

they'll verify your details and solve the issue.

**Reply by LiteBrite82** [reply]
Hi @lutfullahillabib ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by LiteBrite82** [comment]
@lutfullahillabib 
I would highly recommend removing your personal information (phone number, email, real name) from the post. Its a major security risk.
